website/docs/language/functions/templatestring.mdx

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
The documentation here seems quite minimal. It may be difficult for new users to consume this.

Reply:
commit [4fb1950](https://github.com/opentofu/opentofu/pull/1223/commits/4fb1950bb377f66c61af3af40afb85f8d11d343a) resolves this